lambda-runtime/src/AWS/Lambda/Runtime.hs

```diff
@@ -0,0 +1,47 @@
+module AWS.Lambda.Runtime
+  ( module Exports
+  , Client.InternalLambdaClientError (..)
+  , run
+  )
+where
+
+
+import AWS.Lambda.Env             as Exports
```

Comment:
__comment:__ Not entirely sure this `as Exports` is aligned as expected. TBH I don't know if you intended to align it with the code below or not.

Reply:
I intended to align it with the qualified imports below, but now I think I should not. I failed to find a precedent for this.